styles/flow/mod.css

```diff
@@ -49,7 +54,13 @@ blockquote > cite:last-child::before {
 figure {
   display: flex;
   flex-wrap: wrap;
-  justify-content: space-around;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto 1.5rem;
+  padding: 1rem;
+  border-radius: var(--bd-radius);
+  background: var(--bg-muted);
+  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.1);
```

Comment:
This looks okay, but I guess this is the kind of style change that should be voted by the community
The `box-shadow` should inherit the `--shadow` property